Core/Graphics/Source/Graphics.cpp

```diff
@@ -64,12 +64,23 @@ namespace Babylon
 
     void Graphics::Impl::Resize(size_t width, size_t height)
     {
-        std::scoped_lock lock{m_bgfxState.Mutex};
-        m_bgfxState.Dirty = true;
+        {
+            std::scoped_lock lock(m_resolution.Mutex);
+            m_resolution.width = width;
+            m_resolution.height = height;
+        }
+        Resize();
+    }
 
+    void Graphics::Impl::Resize()
```

Comment:
This name confuses me a bit, I think because it's a private helper method overloading a public contract method. I think this method might disappear/change substantially in refactors related to other comments, but if it doesn't, it might be clearer to name it `UpdateBgfxResolution`, or something like that, to clarify its distinction as just an internal helper.

Reply:
makes sense, functions shouldn't be overloaded without good reason.